### PR TITLE
[6.2.z] KatelloAgent: Use internal/CDN Tools appropriately

### PR DESCRIPTION
**Pull request by lpramuk**
- add new settings: cdn, capsule_repo, sattools_repo
- use internal/CDN Tools according to `cdn` setting

(addresses #4429 for 6.2.z)

### CODE REVIEW

**Comment by lpramuk** [comment]
6.2.z cherrypick of PR #4439

**Comment by lpramuk** [comment]
Guys I prefer not to squash 2 commits

File: robottelo/config/settings.py [hunk 1]
```diff
@@ -610,11 +610,14 @@ def __init__(self):
         self._configured = False
         self._validation_errors = []
         self.browser = None
+        self.cdn = None
         self.locale = None
         self.project = None
         self.reader = None
         self.rhel6_repo = None
         self.rhel7_repo = None
+        self.capsule_repo = None
```
Comment (kbidarkar) [inline robottelo/config/settings.py]: Just realized, we may need to move this up for lexicon order.

File: robottelo/config/settings.py [hunk 2]
```diff
@@ -691,10 +694,14 @@ def _read_robottelo_settings(self):
         """Read Robottelo's general settings."""
         self.browser = self.reader.get(
             'robottelo', 'browser', 'selenium')
+        self.cdn = self.reader.get('robottelo', 'cdn', True, bool)
         self.locale = self.reader.get('robottelo', 'locale', 'en_US.UTF-8')
         self.project = self.reader.get('robottelo', 'project', 'sat')
         self.rhel6_repo = self.reader.get('robottelo', 'rhel6_repo', None)
         self.rhel7_repo = self.reader.get('robottelo', 'rhel7_repo', None)
+        self.capsule_repo = self.reader.get('robottelo', 'capsule_repo', None)
```
Comment (kbidarkar) [inline robottelo/config/settings.py]: Same, let's move this up for lexicon order.

Comment (lpramuk) [inline robottelo/config/settings.py]: already merged in master in this order

File: robottelo/config/settings.py (hunk 1 above)
Comment (lpramuk) [inline robottelo/config/settings.py]: lexicon order is breached already anyway

File: robottelo/config/settings.py (hunk 2 above)
Comment (lpramuk) [inline robottelo/config/settings.py]: lexicon order is breached already anyway

**Pull request merged**